CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to CleverHans
+
```

Comment:
Maybe we should have a message saying that if you publish a new attack or defense you're highly encouraged to add it to CleverHans so people can evaluate it fairly?